scipy/optimize/_linprog.py

```diff
@@ -460,12 +460,27 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     >>> x1_bounds = (-3, None)
     >>> from scipy.optimize import linprog
     >>> res = linprog(c, A_ub=A, b_ub=b, bounds=[x0_bounds, x1_bounds])
+    >>> print(res)
+         con: array([], dtype=float64)
+         fun: -21.99999984082494 # may vary
+     message: 'Optimization terminated successfully.'
+         nit: 6 # may vary
+       slack: array([3.89999997e+01, 8.46872439e-08] # may vary
+      status: 0
+     success: True
+           x: array([ 9.99999989, -2.99999999]) # may vary
+
+    Note that the default method for `linprog` is 'interior-point', which is
+    approximate by nature. If you need greater accuracy, try 'revised simplex'.
+
+    >>> res = linprog(c, A_ub=A, b_ub=b, bounds=[x0_bounds, x1_bounds],
+                      method = 'revised simplex')
```

Comment:
This should be ``method='revised simplex'`` *not* ``method = 'revised simplex'`` (note the extra spaces).

Reply:
Right.